### PR TITLE
Coverage following advice from Matt Seddon

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: src/test/index.ts [hunk 1]
```diff
@@ -2,7 +2,32 @@ import * as path from 'path';
 import * as Mocha from 'mocha';
 import * as glob from 'glob';
 
+function setupNyc() {
+	const NYC = require('nyc');
+	const defaultExclude = require('@istanbuljs/schema/default-exclude');
+	const cwd = path.join(__dirname, '..', '..', '..');
```
Comment (mattseddon) [inline src/test/index.ts:8]: [F] This is probably incorrect

File: src/test/index.ts [hunk 2]
```diff
@@ -2,7 +2,32 @@ import * as path from 'path';
...
+	  cwd,
+	  exclude: [...defaultExclude, '**/.vscode-test/**'],
+	  extensions: ['ts'],
+	  hookRequire: true,
+	  hookRunInContext: true,
+	  hookRunInThisContext: true,
+	  instrument: true,
+	  reporter: ['text', 'html'],
+	  sourceMap: true,
+	  tempDirectory: path.join(cwd, 'coverage', 'integration')
```
Comment (mattseddon) [inline src/test/index.ts:21]: [F] This too, we have a jest test suite that runs so we need to run multiple coverage reports into different folders and then merge them. I think (not sure) that you might be able to drop this line.

File: src/test/index.ts (hunk 1 above)
Comment (apupier) [reply src/test/index.ts:8]: I added a console log locally. i tis giving me  `/home/apupier/git/camel-lsp-client-vscode` which is the root folder of the whole extension.

**Comment by apupier** [comment]
There was proof of concept done using c8 on other projects and i still do not plan to look more on this one. So closing.

Thank you very much for your help Matt

**Pull request closed without merging**